docker/docker-compose.yml

```diff
@@ -42,8 +49,10 @@ services:
         # note : To work successfully, it needs pinot to be healthy. This has started parallel for
         # startup experience.
         condition: service_started
+    links:
```

Comment:
why do we need this?

Reply:
While debugging I found in the pas that this service was complaining about zookeeper being unknown. Maybe due to docker version or something else. If I get this green then I will remove it.

Reply:
please do as it is green now!